sdv/constraints/tabular.py

```diff
@@ -531,6 +541,11 @@ def reverse_transform(self, table_data):
             original_column = self._value - diff_column
 
         table_data[self._column_name] = pd.Series(original_column).astype(self._dtype)
+        if self._is_datetime and self._datetime_format:
```

Comment:
is there ever a case where `self._datetime_format` is True, but `self._is_datetime` isn't? There shouldn't be right because the former is only set if the latter is True

Reply:
If `self._is_datetime` is `True` but we are not able to `detect` the format of the date time, then we can't format with `None`.